packages/databases-collections/src/stores/rename-collection.ts

```diff
@@ -24,7 +24,17 @@ export function activateRenameCollectionPlugin(
   );
 
   const onRenameCollection = (ns: { database: string; collection: string }) => {
-    store.dispatch(open(ns.database, ns.collection));
+    dataService
+      .listCollections(ns.database)
```

Comment:
We usually don't run list manually in plugins, the actual logic of listing also requires checking the user privileges, so all together it's pretty expesive for every plugin to list collections on its own. You can use instance model for the purpose of getting a list of collections for a certain database